gradle/libs.versions.toml

```
@@ -0,0 +1,51 @@
+[versions]
+hypertrace-grpcUtils = "0.9.0"
+
+guice = "5.1.0"
+grpc = "1.48.0"
+jetty = "9.4.48.v20220622"
```

might as well bump this to latest too
```suggestion
jetty = "9.4.49.v20220914"
```